src/main/scala/org/broadinstitute/dsde/snoop/ws/SnoopApiService.scala

```diff
@@ -17,21 +23,54 @@ object SnoopApiServiceActor {
   }
 }
 
-class SnoopApiServiceActor(override val executionServiceHandler: RequestContext => WorkflowExecutionService) extends Actor with SnoopApiService {
+class SnoopApiServiceActor(override val executionServiceHandler: RequestContext => WorkflowExecutionService) extends Actor with BaseSnoopApiService with WorkflowExecutionApiService {
+  implicit def executionContext = actorRefFactory.dispatcher
   def actorRefFactory = context
-  def receive = runRoute(snoopRoute)
-}
+  def possibleRoutes = baseRoute ~ workflowRoutes ~ swaggerService.routes 
 
+  def receive = runRoute(possibleRoutes)
 
-// this trait defines our service behavior independently from the service actor
-trait SnoopApiService extends HttpService {
-  implicit def executionContext = actorRefFactory.dispatcher
+  private val snoopConfig: Config = ConfigFactory.parseFile(new File("/etc/snoop.conf"))
```

Comment:
I don't want to be looking up config here. Please pass the config in via the constructor and lookup the config in Boot.scala. You can pass in a swaggerConfig object that I would suggest be a case class constructed from Config loaded in Boot.